targets/targets.json

```diff
@@ -4929,10 +4941,22 @@
             "USTICKER"
         ],
         "device_name": "MAX32625",
-        "release_versions": [
-            "5"
-        ],
-        "public": false
+        "public": false,
+        "supported_application_profiles" : ["full", "bare-metal"],
+        "supported_c_libs": {
+            "arm": [
+                "std",
+                "small"
+            ],
+            "gcc_arm": [
+                "std",
+                "small"
+            ],
+            "iar": [
+                "std"
+            ]
+        },
+        "c_lib" : "small"
```

Comment:
As the target is specified to support the full profile, it is preferable to have the default C library to be the standard one. This is because the small C libraries (Microlib and newlib-nanolib) are not thread-safe and thus cannot be used with the full profile.

Reply:
Thanks for review @hugueskamba. Done.